app/controllers/shipit/tasks_controller.rb

```diff
@@ -25,7 +25,7 @@ def create
       @definition = stack.find_task_definition(params[:definition_id])
 
       if @definition.allow_concurrency? || params[:force] || !@stack.active_task?
-        @task = stack.trigger_task(params[:definition_id], current_user)
+        @task = stack.trigger_task(params[:definition_id], current_user, env: task_params[:env])
```

Comment:
I had missed this embarrassingly enough, so the app controller wasn't respecting the variables through the UI :panda_face: